tests/suites/test_suite_ssl.function

```diff
@@ -4200,6 +4200,10 @@ void tls13_write_early_data(int scenario)
             break;
 
         case TEST_EARLY_DATA_HRR:
+            /*
+             * Remove server support for the group negotiated in
+             * mbedtls_test_get_tls13_ticket() forcing an HelloRetryRequest.
```

Comment:
(nit) it's usually `a hello...` (the canonical English word for using `an` in front of a consonant is `historic`, although some people still use `a historic ...`, while in speech it's nearly always `an historic ...` since the `h` is very weak)